extensions/ui/org.eclipse.smarthome.ui.paper/tests/configuration.binding.test.js

```diff
@@ -0,0 +1,71 @@
+describe('module PaperUI.controllers.configuration.bindings', function() {
+    beforeEach(function() {
+        module('PaperUI');
+    });
+    var restConfig;
```

Comment:
I´m only talking about `var restConfig` since it is only assigned in the nested `beforeEach` block in line 28. So the definition can be done in line 24 which will narrow down the scope of `restConfig`.